### DISCUSSION TITLE
if you use  markoshust/magento-php:8.1-fpm-0 you will not get Node and Npm installed.

### DISCUSSION THREAD

**Discussion by JesperHerrloff** [markshust/docker-magento#1025] [General]
Solution is to use  markoshust/magento-php:8.1-fpm-1 or  markoshust/magento-php:8.1-fpm or  markoshust/magento-php:8.1-fpm-develop

I cant understand how I manage to find the most fascinating things with every project. I might start to work as a QA Tester or something like that. Everything seems to have some small things that I discover. 

Keep the good work up.

**Comment by YevhenZvieriev** [comment]
Hi, @JesperHerrloff 

Thank you for bringing this to our attention. After further investigation, it seems that the issue you raised has already been addressed in a commit made 2 years ago. The commit added Node.js and npm to the markoshust/magento-php:8.1-fpm image.

You can review the specific commit [here](https://github.com/markshust/docker-magento/commit/600dd50a8ecd800370c38d03de5efcac7e2ccad9).

As you said, to resolve the current issue, you should using one of the following images:

- markoshust/magento-php:8.1-fpm-1
- markoshust/magento-php:8.1-fpm
- markoshust/magento-php:8.1-fpm-develop

Please note that markoshust/magento-php:7.4-fpm-14 also does not include Node.js and npm, so I recommend using markoshust/magento-php:7.4-fpm-15
